bedrock/firefox/views.py

```diff
@@ -308,28 +227,6 @@ def firefox_os_geo_redirect(request):
     return HttpResponseRedirect(reverse('firefox.os.ver.{0}'.format(version)))
 
 
-@csrf_protect
-def firefox_partners(request):
-    # If the current locale isn't in our list, return the en-US value
-    press_locale = request.locale if (
-        request.locale in LOCALE_FXOS_HEADLINES) else 'en-US'
-
-    template_vars = {
-        'locale_headline_url': LOCALE_FXOS_HEADLINES[press_locale]['url'],
-        'locale_headline_title': LOCALE_FXOS_HEADLINES[press_locale]['title'],
-        'js_common': JS_COMMON,
-        'js_mobile': JS_MOBILE,
-        'js_desktop': JS_DESKTOP,
-    }
-
-    form_kwargs = {
-        'interest_set': 'fx',
-        'lead_source': 'www.mozilla.org/firefox/partners/'}
```

Comment:
With the removal of this page, the [custom interest set](https://github.com/mozilla/bedrock/blob/master/bedrock/mozorg/forms.py#L253) is no longer in use. I feel it's a good thing to retain the ability to easily change the interests displayed, but I'm not exactly sure what we should do (if anything?) with the `WebToLeadForm` code at this point. [Here's one take on what we could do](https://gist.github.com/jpetto/7d4dde2790c9dbf50d1a).

Thoughts? Maybe @pmclanahan wants to weigh in?